### PR TITLE
Added link to player's profile in all appropriate notifications

### PR DESCRIPTION
**Pull request by OsamahAnsari**
Added a link to the player's profile wherever a player's name shows up in a notification

Handles player profile links for the following:
    - Group broadcasts
    - Comments of all types (news, puzzles, group, etc.)
    - Direct private messages and replies to them

Resolves: #223 
Combined with #236, should fix all issues related to notifications that remain. 

**New notification view after #236 and this patch:**
Notifications Player Profile Links

### CODE REVIEW

File: src/views/feed/ActivityFeed/components/CommentMessageItem.vue [hunk 1]
```diff
@@ -7,7 +7,10 @@
       :message="message.content.body"
     >
       <template>
-          {{ notification.target2_name + ' ' }} {{ $t('activity-feed:commented-on') + ' ' }}
+          <router-link :to="`/players/` + message.sender">
+            {{ notification.target2_name + ' ' }} 
+          </router-link>
+          {{ $t('activity-feed:commented-on') + ' ' }}
```
Comment (luxaritas) [inline src/views/feed/ActivityFeed/components/CommentMessageItem.vue:13]:
```suggestion
            {{ notification.target2_name }} 
          </router-link>
          {{ ' ' + $t('activity-feed:commented-on') + ' ' }}
```

File: src/views/feed/ActivityFeed/components/GroupMessageItem.vue [hunk 2]
```diff
@@ -7,7 +7,10 @@
       :avatar="avatar"
     >
       <template>
-          {{ notification.target2_name + ' ' }} {{ isInvite ? $t('activity-feed:invite') : $t('activity-feed:broadcast') + ' ' }}
+          <router-link :to="`/players/` + message.sender">
+            {{ notification.target2_name + ' ' }} 
+          </router-link>
+          {{ isInvite ? $t('activity-feed:invite') : $t('activity-feed:broadcast') + ' ' }}
```
Comment (luxaritas) [inline src/views/feed/ActivityFeed/components/GroupMessageItem.vue:13]:
```suggestion
          <router-link :to="`/players/` + message.sender">
            {{ notification.target2_name }} 
          </router-link>
          {{ ' ' + isInvite ? $t('activity-feed:invite') : $t('activity-feed:broadcast') + ' ' }}
```

**Review by luxaritas — CHANGES_REQUESTED** [review]
Couple minor tweaks

File: src/views/feed/ActivityFeed/components/GroupMessageItem.vue (hunk 2 above)
Comment (OsamahAnsari) [reply src/views/feed/ActivityFeed/components/GroupMessageItem.vue]: Done, but slightly needed to change to make the ternary operator work as intended,

File: src/views/feed/ActivityFeed/components/CommentMessageItem.vue (hunk 1 above)
Comment (OsamahAnsari) [reply src/views/feed/ActivityFeed/components/CommentMessageItem.vue]: Done

**Pull request merged**